ci/deploy-forklift.sh

```diff
@@ -56,9 +60,25 @@ EOF
 
 # --------------------
 
+echo ""
+echo "Installing VolumePopulator CRD"
+echo "=============================="
+
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-csi/volume-data-source-validator/v1.0.1/client/config/crd/populator.storage.k8s.io_volumepopulators.yaml
```

Comment:
do we need it for local dev env?

Reply:
it's a requirement fro forklift, so we need it for forklift to work